Source/Core/DolphinWX/FrameTools.cpp

```diff
@@ -292,6 +293,70 @@ void CFrame::OpenGeneralConfiguration(wxWindowID tab_id)
   m_main_config_dialog->SetFocus();
 }
 
+void CFrame::CycleProfile(CycleDirection cycle_direction)
+{
+  // load the hotkey config
+  InputConfig* hk_conf = HotkeyManagerEmu::GetConfig();
+  auto* hotkey_controller = hk_conf->GetController(0);
+
+  // load the wiimote config
+  InputConfig* wm_conf = Wiimote::GetConfig();
+  std::string profile_name(File::GetUserPath(D_CONFIG_IDX));
+
+  // find all profiles
+  profile_name += "Profiles/" + wm_conf->GetProfileName();
+  std::vector<std::string> sv = Common::DoFileSearch({profile_name}, {".ini"});
+
+  // remove all profiles that do not fit the filter string
+  if (!hotkey_controller->profile_filter.empty())
+  {
+    sv.erase(std::remove_if(sv.begin(), sv.end(),
+                            [filter = hotkey_controller->profile_filter](const std::string& s) {
+                              return s.find(filter) == std::string::npos;
+                            }),
+             sv.end());
+  }
+
+  // check if there are any profiles to cycle through
+  if (sv.empty())
+  {
+    if (hotkey_controller->profile_filter.empty())
```

Comment:
Both the if and else case here should be braced since the body of the else should be braced if it travels across more than one line; the if being braced, since we avoid inconsistent bracing of paths. (missed this in my initial review, sorry).